pkg/checks/latency/latency.go

```diff
@@ -71,9 +71,11 @@ type result struct {
 
 // metrics defines the metric collectors of the latency check
 type metrics struct {
-	duration  *prometheus.GaugeVec
-	count     *prometheus.CounterVec
-	histogram *prometheus.HistogramVec
+	duration      *prometheus.GaugeVec
+	totalDuration *prometheus.GaugeVec
```

Comment:
If we're renaming the old metrics we might as well throw em out directly. No need to add more complexity if we're not doing a deprecation. I would just delete the old metrics, so we don't carry them around and forget to delete them.

Reply:
Yes, I go with you as discussed. Status code should not be the scope of the latency check in the future. I have added a depreciation flag for now.

Add. I have rm the new counter as well.